includes/admin/forms/class-metabox-form-data.php

```diff
@@ -174,6 +174,7 @@ function get_settings() {
 								'description'   => __( 'This text appears as a label below the custom amount field for set donation forms. For multi-level forms the text will appear as it\'s own level (ie button, radio, or select option).', 'give' ),
 								'id'            => $prefix . 'custom_amount_text',
 								'type'          => 'text_medium',
+								'default'       => __( 'Custom Amount', 'give' ),
 								'attributes'    => array(
 									'rows'        => 3,
 									'placeholder' => __( 'Give a Custom Amount', 'give' ),
```

Comment:
This needs to be updated to match the default

Reply:
I removed it, we do not need this anymore.

Reply:
ref: https://github.com/impress-org/givewp/commit/0e2d954f16346cd85543076abba58485d6b37983